controllers/admin/AdminDPDBalticsShipmentSettingsController.php

```diff
@@ -187,6 +187,13 @@ protected function initOptions()
                         'cast' => 'intval',
                         'type' => 'bool',
                     ],
+                    Config::AUTOMATED_PARCEL_RETURN => [
+                        'title' => $this->l('Automated parcel return'),
```

Comment:
Maybe name could be something like Automatically generate return label P.S Translations has to be added aswell

Reply:
Done